modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -88,22 +112,30 @@ final case class GHOSTController[F[_]: Sync](ghostClient: GHOSTClient[F],
       _ <- SeqActionF.apply(Log.debug("Completed GHOST configuration"))
     } yield ()
 
+  // We use a dummy observation for now, since at this point, we cannot actually observe using the instrument.
   def observe(fileId: ImageFileId, expTime: Time): SeqActionF[F, ImageFileId] =
-    EitherT(ghostClient.observe(fileId, expTime.toMilliseconds.milliseconds).map(_ => fileId).attempt)
-      .leftMap {
-        case CommandResultException(_, "Message cannot be null") => Execution("Unhandled observe command")
-        case CommandResultException(_, m)                        => Execution(m)
-        case f                                                   => SeqexecException(f)
-      }
+    SeqActionF.apply((fileId, expTime)._1) // suppress unused error
+  //    EitherT(ghostClient.observe(fileId, expTime.toMilliseconds.milliseconds).map(_ => fileId).attempt)
+  //      .leftMap {
+  //        case CommandResultException(_, "Message cannot be null") => Execution("Unhandled observe command")
+  //        case CommandResultException(_, m)                        => Execution(m)
+  //        case f                                                   => SeqexecException(f)
+  //      }
 
   def endObserve: SeqActionF[F, Unit] = SeqActionF.void
 }
 
 object GHOSTController {
-  sealed abstract class BundleConfig(val configName: String)
+  sealed abstract class BundleConfig(val configName: String) {
+    def determineType(t: IFUTargetType): BundleConfig = t match {
+      case IFUTargetType.SkyPosition => BundleConfig.Sky
+      case _                         => this
+    }
+  }
   object BundleConfig {
-    case object Standard extends BundleConfig(configName = "lo")
-    case object HighRes extends BundleConfig(configName = "hi")
+    case object Standard extends BundleConfig(configName = "IFU_LORES")
+    case object HighRes  extends BundleConfig(configName = "IFU_HRES")
+    case object Sky      extends BundleConfig(configName = "IFU_SKY")
   }
 
   sealed abstract class IFUNum(val ifuNum: Int) {
```

Comment:
Why `abstract class` rather than `sealed trait`?

Reply:
If I use `sealed trait`, I can't pass in the `val ifuNum: Int` parameter, can I?

The compiler complains in `IFU1` and `IFU2` about reassignment to `val`.

Reply:
Hmmm... I'd still have to change `IFU1` and `IFU2` to then have these as val params instead of passing them up to the superclass. Is there an advantage to this, out of curiosity?

Reply:
i.e.:
```
 sealed trait IFUNum {
    val ifuNum: Int
    val ifuStr: String = s"ghost:cc:cu:ifu$ifuNum"
  }
  object IFUNum {
    case object IFU1 extends IFUNum {
        override val ifuNum = 1
    }
    case object IFU2 extends IFUNum {
        override val ifuNum = 2
    }
  }
```

Reply:
TBH there are only details in one case v/s the other. I often fall back to sealed trait and use `abstract class` only on some cases. perhaps @tpolecat has a better explanation

You can leave it as is, it probably doesn't make a big difference